### PR TITLE
THRIFT-3905 Initialize bool, int, and double properties

### PR DESCRIPTION
**Pull request by markerickson-wf**
see https://issues.apache.org/jira/browse/THRIFT-3905

@tylertreat-wf 
@stevenosborne-wf 
@brianshannan-wf

### CODE REVIEW

**Comment by brianshannan-wf** [comment]
+1

File: compiler/cpp/src/generate/t_dart_generator.cc [hunk 1]
```diff
@@ -2312,6 +2313,34 @@ string t_dart_generator::type_to_enum(t_type* type) {
   throw "INVALID TYPE IN type_to_enum: " + type->get_name();
 }
 
+std::string t_dart_generator::init_value(t_type* ttype) {
+	string result = "";
+	if (ttype->is_typedef()) {
+  	ttype = ((t_typedef*)ttype)->get_type();
```
Comment (tylertreat-wf) [inline compiler/cpp/src/generate/t_dart_generator.cc]: nit: indentation?

**Comment by tylertreat-wf** [comment]
lgtm

File: compiler/cpp/src/generate/t_dart_generator.cc (hunk 1 above)
Comment (markerickson-wf) [inline compiler/cpp/src/generate/t_dart_generator.cc]: ugh, a tab vs space issue - thanks

**Comment by brianshannan-wf** [comment]
+1

**Pull request closed without merging**